nautobot/extras/models/jobs.py

```diff
@@ -807,6 +816,7 @@ def enqueue_job(
         task_queue=None,
         job_result=None,
         synchronous=False,
+        ignore_singleton_lock=False,
```

Comment:
Do we need to similarly enhance the `create_schedule` method to allow creating scheduled jobs that ignore the singleton?

Reply:
I just tested it manually and it works both way, i.e. the job fails as expected when one is already running and correctly overrides the lock if the checkbox is set. This appears to happen because this option is passed along via the `kwargs` field on `ScheduledJob`. I am happy to change it such that it uses `celery_kwargs` instead if desired, I just wanted to check in with you because as of now it already work.

```
>>> sj = ScheduledJob.objects.get(name="TEST")
>>> sj.celery_kwargs
{'queue': None, 'nautobot_job_profile': False}
>>> sj.kwargs
{'_ignore_singleton_lock': True}
```

Reply:
Oh, I guess `kwargs` is supposed to be the job arguments that get passed to `run`, is that accurate?

Reply:
Yeah... it looks like keys in `kwargs` that don't match the Job's input variables get discarded by `deserialize_data()` so I think that's why it's working presently. But it might be cleaner to follow the pattern of `nautobot_job_profile` and store it in the `celery_kwargs` instead?

Reply:
Added as per discussion.